pkg/api/v1beta3/types.go

```diff
@@ -200,26 +200,26 @@ type VolumeSource struct {
 	// to see the host machine. Most containers will NOT need this.
```

Comment:
Users have to specify either name or generateName, in post, put, and patch, and it will be returned by get.